CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ set(TARAXA_DB_MAJOR_VERSION 2)
 # Minor version should be modified when changes to the database are made in the tables that can be rebuilt from the
 # basic tables
 set(TARAXA_DB_MINOR_VERSION 3)
+# If config params are changed we should increase the version
+set(TARAXA_CONFIG_VERSION 1)
```

Comment:
why dont we use node version instead of this ?

Reply:
changing of node version does not mean, changing the config. But I guess we can do it like that